snippets/fsharp/VS_Snippets_CLR_System/system.console.class.unsafe/fs/setfont1.fs

```diff
@@ -0,0 +1,80 @@
+﻿// <Snippet3>
+open System
+open System.Runtime.InteropServices
+
+[<LiteralAttribute>]
+let STD_OUTPUT_HANDLE = -11
+[<LiteralAttribute>]
+let TMPF_TRUETYPE = 4
+[<LiteralAttribute>]
+let LF_FACESIZE = 32
+let INVALID_HANDLE_VALUE = IntPtr(-1)
+
+[<Struct>]
+[<StructLayout(LayoutKind.Sequential)>]
+type (*internal*) COORD =
```

Comment:
Is there a reason for the `(* internal *)`?

Reply:
I believe it's just left over from attempts to match the C# example. I'll remove the inline comment.